packages/app/src/embed/components/Files/index.js

```diff
@@ -44,28 +47,21 @@ function Files({
 
   return (
     <Container>
-      {sortBy(childrenDirectories, d => d.title).map(d => (
-        <div key={d.shortid}>
-          <File
-            id={d.id}
-            shortid={d.shortid}
-            title={d.title}
-            type="directory-open"
-            depth={depth}
-            setCurrentModule={setCurrentModule}
-          />
-          <Files
-            modules={modules}
-            directories={directories}
-            directoryId={d.shortid}
-            depth={depth + 1}
+      {sortBy(childrenDirectories, directory => directory.title).map(
+        directory => (
+          <Directory
```

Comment:
Created a new component to make the component easier to understand

Reply:
That's very nice!